meson.build

```diff
@@ -19,6 +19,14 @@ is_freebsd = host_machine.system().startswith('freebsd')
 datadir = get_option('datadir')
 sysconfdir = get_option('sysconfdir')
 prefix = get_option('prefix')
+libexecdir = get_option('libexecdir')
+
+if libexecdir == ''
+  libexecdir = join_paths(prefix, 'lib')
+endif
+rundir = join_paths(libexecdir, 'sway')
+add_project_arguments('-DRUNDIR="/@0@"'.format(rundir), language : 'c')
```

Comment:
Why is this defined as RUNDIR? Just call it LIBEXECDIR everywhere. Be consistent.